### PR TITLE
Dx 1682

### PR DESCRIPTION
**Pull request by ajrice6713**

### CODE REVIEW

File: public/index.php [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        if($call_id == $dValue)
+          array_splice($data['callIds'], $key, 1);
+      }
+    }
+  }
+  $newJsonString = $data;
+  // var_dump($newJsonString);
+  file_put_contents($file, json_encode($newJsonString));
+  return json_encode($newJsonString);
+  // return $data;
```
Comment (jmulford-bw) [inline public/index.php]: Could probably get rid of these comments

File: public/index.php [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+      $arr = modifyArray($call_id_file, $args['id'], 'remove');
+      $response = $response->withStatus(200)->withHeader('Content-Type', 'application/xml');
+      $response->getBody()->write($arr);
+      return $response;
+    } catch (BandwidthLib\APIException $e) {
+      $response = $response->withStatus(404);
+      $response->getBody()->write($e);
+      return $response;
+    }
+});
```
Comment (jmulford-bw) [inline public/index.php]: You could maybe clean this up by checking the existence of the ID before doing an API request. That way you can return a 404 on a call not found before making an API request, then do the API request and handle the error separately

**Review by jmulford-bw — CHANGES_REQUESTED** [review]
1 small thing, otherwise 👍

File: public/index.php (hunk 2 above)
Comment (ajrice6713) [reply public/index.php]: good idea - done

File: public/index.php [hunk 3]
```diff
@@ -0,0 +1,131 @@
+<?php
+
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Slim\Factory\AppFactory;
+
+require __DIR__ . '/../vendor/autoload.php';
+
+$BANDWIDTH_ACCOUNT_ID = getenv("BANDWIDTH_ACCOUNT_ID");
+$BANDWIDTH_USERNANME = getenv("BANDWIDTH_USERNAME");
```
Comment (hamermike) [inline public/index.php]:
```suggestion
$BANDWIDTH_USERNAME = getenv("BANDWIDTH_USERNAME");
```

File: public/index.php [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+$BANDWIDTH_PASSWORD = getenv("BANDWIDTH_PASSWORD");
+$BANDWIDTH_VOICE_APPLICATION_ID = getenv("BANDWIDTH_VOICE_APPLICATION_ID");
+$BASE_URL = getenv("BASE_URL");
+
+$config = new BandwidthLib\Configuration(
+    array(
+        "voiceBasicAuthUserName" => $BANDWIDTH_USERNANME,
```
Comment (hamermike) [inline public/index.php]:
```suggestion
        "voiceBasicAuthUserName" => $BANDWIDTH_USERNAME,
```

**Review by hamermike — CHANGES_REQUESTED** [review]

**Review by hamermike — APPROVED** [review]

**Pull request merged**